### ISSUE TITLE
The name of file is different than the main function in it

### ISSUE THREAD

**Issue by it-praktyk** [trondhindenes/AnsibleDscModuleGenerator#10]
The file name: AnsibleWinModuleGen.ps1
The function name: Invoke-AnsibleWinModuleGen

I think that a good practice is 
1. save one function per file (a module need to be psd1 based for that)
2. the file name equal to the main function name

What do you think about the name: ConvertTo-AnsibleModule for the main function ?

**Comment by trondhindenes** [comment]
Agreed. I'll make this into a proper module with a module manifest.
